.readthedocs.yml

```diff
@@ -2,13 +2,14 @@ version: 2
 
 sphinx:
   configuration: docs/source/conf.py
+  fail_on_warning: false
```

Comment:
Just curious, what kind of warnings we want to ignore?

Reply:
There are warnings being emitted because of a bug with the MyST notebook parser: https://github.com/executablebooks/MyST-NB/issues/464

Essentially, any notebooks that depend on Holoviz (ie: interactable widgets) are not understoud when building the documentation. Aside from adding a way to squelch the warnings in `conf.py`, if we don't set this to false, none of our builds on ReadTheDocs will resolve.

Not sure how best to move forward here.

Reply:
It's fine to ignore warnings, as long as we know these are safe to ignore (no silent failure).